frontend/src/views/WheresMyGene/components/CellInfoSideBar/index.tsx

```diff
@@ -58,14 +58,22 @@ function CellInfoSideBar({
 
   const [hoverStartTime, setHoverStartTime] = useState(0);
 
-  const useHandleHoverEnd = useCallback((event: EVENTS, payload = {}) => {
-    if (Date.now() - hoverStartTime > 2 * 1000) {
-      track(event, payload);
-    }
-  }, [hoverStartTime])
+  const useHandleHoverEnd = (event: EVENTS, payload = {}) => {
+    return useCallback(() => {
+      if (Date.now() - hoverStartTime > 2 * 1000) {
+        track(event, payload);
+      }
+    }, [hoverStartTime]);
+  };
```

Comment:
Does event and payload need to be added to the dependency list for `useCallback`?

Reply:
no i dont think so. when the function is called, it returns a fresh callback each time, so we never have a situation where the event and payload change but the callback doesn't.

Reply:
Wait, what's the point of useCallback here? We're circumventing the memoization that is serves.